src/PlanetParade.vue

```diff
@@ -349,18 +382,43 @@
                 </p>
-                <h4>What is the significance of the planet parade?</h4>
+                <h3>How do I find my way around the sky?</h3>
+                <p>
+                  Here are some tips for finding the planets in the sky from the Northern Hemisphere.
+                  <ul>
+                    <li>
+                      Venus is the brightest planet and should be easiest to spot above the western horizon.
+                    </li>
+                    <li>
+                      Once you've found Venus, look down toward the horizon to find Saturn.
+                    </li>
+                    <li>
+                       Imagine a line between Saturn and Venus that points along a big arc towards Jupiter (which is above and to the right of Orion).
+                    </li>
+                    <li> 
+                      Keep following that path to a bright, red object, which is Mars.
+                    </li>
+                    <li>
+                      Mercury's position will change throughout the month, but it will be near the horizon, close to Venus and Saturn.
+                    </li>
+                  </ul> 
+                </p>
+                <h3>What is the significance of the planet parade?</h3>
                 <p>
                   The planet parade happens when all the planets happen to be on the same side of their orbits around the Sun as Earth, so they all are on Earth's night-time side. (When a planet is on the opposite side of the Sun during Earth, that means it is up in the sky during the day when the Sun outshines it.) 
                 </p>
                 <p>
                   The planets are also not really all in a line. It only appears that way from Earth because Earth and the other planets all orbit the Sun in roughly the same plane (as if they were on a vinyl record). From Earth's point of view, all the other solar system objects appear to move along a circular path around the sky, which ancient astronomers called the "ecliptic." Check <span style="color: var(--accent-color)">Ecliptic</span> in the control box to display this path in the virtual sky. 
                 </p>
-                <h4>Will the planets be this big in my sky?</h4>
+                <h3>What else can I do to learn more?</h3>
+                <p>
+                  Take your own images of the planets using <a href="https://mo-www.cfa.harvard.edu/OWN/todays_solar_system.html#SS" target="_blank" rel="noopener noreferrer">MicroObservatory's Observing With NASA</a> program!
+                </p>
+                <h3>Will the planets be this big in my sky?</h3>
                 <p>
                   Unfortunately, no. We scaled up the size of the planet images so you would be able to see them when zoomed out to a large swath of sky. In reality, the planets will appear in the sky as very small discs or points of light.
```

Comment:
I would just say points of light.

Reply:
Yeah, that's what I had at first, but I was worried that hard core people might quibble about that because one of the things that distinguishes planets from stars is that they are not points. How about: "points of light (or very small discs through binoculars or a telescope)."

Reply:
Yeah, that is better. I guess my assumption was people looking with just their eyes